### ISSUE TITLE
v12: SQL upgrade script fails because of tenant script

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#7133]
**Describe the bug**
Error:
```
sh: 1: /usr/local/pf/db/upgrade-tenant-11.2-12.0.pl: Permission denied
ERROR at line 55: Failed to open file '/usr/local/pf/db/upgrade-tenant-11.2-12.0.sql', error: 2
```

Full output
```
root@test-fresh-lt:/usr/local/pf# mysql -v pf < db/upgrade-X.X-X.Y.sql
--------------
SET @MAJOR_VERSION = 11
--------------

--------------
SET @MINOR_VERSION = 3
--------------

--------------
SET @PREV_MAJOR_VERSION = 11
--------------

--------------
SET @PREV_MINOR_VERSION = 2
--------------

--------------
SET @VERSION_INT = @MAJOR_VERSION << 16 | @MINOR_VERSION << 8
--------------

--------------
SET @PREV_VERSION_INT = @PREV_MAJOR_VERSION << 16 | @PREV_MINOR_VERSION << 8
--------------

--------------
DROP PROCEDURE IF EXISTS ValidateVersion
--------------

--------------
CREATE PROCEDURE ValidateVersion()
BEGIN
    DECLARE PREVIOUS_VERSION int(11);
    DECLARE PREVIOUS_VERSION_STRING varchar(11);
    DECLARE _message varchar(255);
    SELECT id, version INTO PREVIOUS_VERSION, PREVIOUS_VERSION_STRING FROM pf_version ORDER BY id DESC LIMIT 1;

      IF PREVIOUS_VERSION != @PREV_VERSION_INT THEN
        SELECT CONCAT('PREVIOUS VERSION ', PREVIOUS_VERSION_STRING, ' DOES NOT MATCH ', CONCAT_WS('.', @PREV_MAJOR_VERSION, @PREV_MINOR_VERSION)) INTO _message;
        SIGNAL SQLSTATE VALUE '99999'
              SET MESSAGE_TEXT = _message;
      END IF;
END
--------------

--------------
ALTER DATABASE
    CHARACTER SET = 'utf8mb4'
    COLLATE = 'utf8mb4_bin'
--------------

Checking PacketFence schema version...
--------------
call ValidateVersion
--------------

sh: 1: /usr/local/pf/db/upgrade-tenant-11.2-12.0.pl: Permission denied
ERROR at line 55: Failed to open file '/usr/local/pf/db/upgrade-tenant-11.2-12.0.sql', error: 2
```

This is on Debian 11

Workaround: `chmod a+x /usr/local/pf/db/upgrade-tenant-11.2-12.0.pl`

**Comment by jrouzierinverse** [comment]
This is a packaging issue on debian.
The script /usr/local/pf/db/upgrade-tenant-11.2-12.0.pl is not executable.
@fdurand, can you look at it?

**Comment by nqb** [comment]
I will.

**Comment by nqb** [comment]
Fix: 714b8d292b
